pkg/service/pipeline.go

```diff
@@ -631,7 +631,7 @@ func (s *service) preTriggerPipeline(ctx context.Context, isAdmin bool, ns resou
 	for {
 		var nsSecrets []*datamodel.Secret
 		// TODO: should use ctx user uid
-		nsSecrets, _, pt, err = s.repository.ListNamespaceSecrets(ctx, ns.Permalink(), 100, "", filtering.Filter{})
+		nsSecrets, _, pt, err = s.repository.ListNamespaceSecrets(ctx, ns.Permalink(), 100, pt, filtering.Filter{})
 		if err != nil {
```

Comment:
NIT: I am not quite sure why we need this part in this PR.
I will trace code more to catch up codebase later.

Reply:
It is just a small bug-fix, I put this in the same PR.